CHANGES.md

```diff
@@ -11,6 +11,15 @@ ones in. -->
 
 --------------------------------------------------------------------------------
 
+## 2.3.0 (<span actions:bind='release-date'>Released 2024-06-18</span>)
+
+Minor changes required for compatibility with cylc-flow 8.3.0 (via cylc-rose).
+
+### Enhancements
+
+[#2754](https://github.com/metomi/rose/pull/2754) - support for file creation from Git repositories
```

Comment:
Formatting as a major heading because of the dashes below
```suggestion
[#2754](https://github.com/metomi/rose/pull/2754) - support for file creation from Git repositories

```